### ISSUE TITLE
The website is down - NXDOMAIN for taskjs.org

### ISSUE THREAD

**Issue by rsp** [mozilla/task.js#43]
The website http://taskjs.org/ is not available:

> This site can’t be reached
> taskjs.org’s server DNS address could not be found.
> DNS_PROBE_FINISHED_NXDOMAIN

It's not a typo in address - the Web Archive shows the previously available website:

* https://web.archive.org/web/20161117122506/http://taskjs.org:80/

Is the URL different now or is the website down temporarily?

**Comment by 0joshuaolson1** [comment]
Someone else owns the site now.

**Comment by rsp** [comment]
@0joshuaolson1 Yes, apparently. I would argue that neither the GitHub description nor the official README should point to a website owned by someone else that currently says: "TASK JS: THE FUTURE OF ONLINE BUSINESS IN A FRENCH-ENGLISH BLOG" if the project is to be considered alive - and if it isn't then maybe explicitly stating so in the documentation would be a better idea.

**Comment by rsp** [comment]
@dherman @BYK @samth Is this project alive at all?

When I submitted this issue almost a year ago the website was down (non-existent domain) but now someone has bought that domain and the README points to a website that says: "TASK JS: THE FUTURE OF ONLINE BUSINESS IN A FRENCH-ENGLISH BLOG"

What is the current address of the Task.js website?
